stdlib/sys.pyi

```diff
@@ -185,19 +299,30 @@ _TraceFunc = Callable[[FrameType, str, Any], Optional[Callable[[FrameType, str,
 def gettrace() -> _TraceFunc | None: ...
 def settrace(tracefunc: _TraceFunc | None) -> None: ...
 
-class _WinVersion(tuple[int, int, int, int, str, int, int, int, int, tuple[int, int, int]]):
-    major: int
-    minor: int
-    build: int
-    platform: int
-    service_pack: str
-    service_pack_minor: int
-    service_pack_major: int
-    suite_mast: int
-    product_type: int
-    platform_version: tuple[int, int, int]
-
 if sys.platform == "win32":
+    # A tuple of length 5, even though it has more than 5 attributes.
+    @final
+    class _WinVersion(_uninstantiable_structseq, tuple[int, int, int, int, str]):
+        @property
+        def major(self) -> int: ...
+        @property
+        def minor(self) -> int: ...
+        @property
+        def build(self) -> int: ...
+        @property
+        def platform(self) -> int: ...
+        @property
+        def service_pack(self) -> str: ...
+        @property
+        def service_pack_minor(self) -> int: ...
+        @property
+        def service_pack_major(self) -> int: ...
+        @property
+        def suite_mast(self) -> int: ...
+        @property
+        def product_type(self) -> int: ...
+        @property
+        def platform_version(self) -> tuple[int, int, int]: ...
     def getwindowsversion() -> _WinVersion: ...
```

Comment:
Note to self: I will have to spin up a windows virtual machine to check these. Will mark resolved when done.

Reply:
Typo: suite_mast --> suite_mask

Reply:
Have fixed the typo (it predated this PR, FWIW)